docs/reference-guides/block-api/block-supports.md

```diff
@@ -53,7 +52,7 @@ Anchors let you link directly to a specific block on a page. This property adds
 ```js
 // Declare support for anchor links.
 supports: {
-	anchor: true
+	anchor: true;
```

Comment:
Is this change intended? It seems like we are making the json invalid.

Reply:
I don't think so. Let me check what's happened here...